script/race.yml

```
@@ -141,3 +141,31 @@
     - allow list
   inconsiderate:
     - whitelist
+- type: simple
+  note: Avoid using terms that imply a group has not changed over time and that they are inferior
+    - simple
```

You’re missing `considerate` here:

```suggestion
  considerate:
    - simple
```

---

Oops, my mistake! Thanks for spotting that